Final_Project_Billinis.qmd

```diff
@@ -74,53 +71,49 @@ The `echo: false` option disables the printing of code (only output is displayed
 ```
 
 ```{r}
-install.packages("ggmap")
+#install.packages("ggmap")
 library(dplyr)
 library(tidyverse)
 library(lubridate)
 library(ggplot2)
-map_data(atlantic)
+library(basemaps)
+
 AerialAtl <- Grships %>% 
   filter(Event == "Aerial Bombing" & Location == "Atlantic")
 head(AerialAtl)
 ```
 
-
 ```{r}
 mapinfo <- map_data(world)
 ```
 
-
 ```{r}
 library(ggplot2)
 library(maps)
 library(tidyverse)
+library(sf) 
```

Comment:
the map_data function previously wouldn't run because it exists in the sf package. Recommend moving all libraries to the top of the .qmd file